GameData/RealSolarSystem/ExtraSettings/RSSnoInclinationPatch.cfg

```diff
@@ -0,0 +1,146 @@
+@Kopernicus:AFTER[RealSolarSystem]
+{
+    %AxialTilt = #$@AxialTiltSettings/AxialTilt$
```

Comment:
This will cause MM to error since the requested node doesn't exist, right? @Sigma88